doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -350,11 +350,15 @@ Bluetooth mesh samples
 
 * :ref:`bluetooth_mesh_light_lc` sample:
 
+  * Added the Kconfig option :kconfig:option:`CONFIG_BT_MESH_COMP_PAGE_2` for Composition Data Page 2 support.
```

Comment:
I think these release notes need to change as per the current changes. Also applies to the similar lines in this file.
```suggestion
  * Added Composition Data Page 2 support.
```